virttest/openvswitch.py

```diff
@@ -356,10 +356,6 @@ def __init__(self, db_path=None, db_socket=None, db_pidfile=None,
         :param ovs_pidfile: Path of OVS ovs-vswitchd pid.
         :param install_prefix: Path where is openvswitch installed.
         """
-        sup = super(man[self.__class__, OpenVSwitchSystem], self)
-        sup.__init__(self, db_path, db_socket, db_pidfile, ovs_pidfile,
-                     dbschema, install_prefix)
```

Comment:
I'm afraid that, by only removing these two lines, the above listed init args wouldn't be set correctly.

It looks like the `VersionableClass` facilities has the problem to handle `__init__()` well in multiple inheritance situation.

Reply:
@luckyh If `OpenVSwitch.__init__()` requires a change, I suppose should be removing those parameters from its init and super.init.
Guess it wants pass the parameters to `OpenVSwitchControl`, but it should never passed to it before, since `OpenVSwitchControl` are not in the super list and it actually calls `object.init`. Where I can see does handle the parameters is `OpenVSwitch.init_new()`.  Or the parameters are initialized when `OpenVSwitchSystem()` called, but not via `init()`, since OpenVSwitchControl redefine its `new()` with those parameters, that should be where the instance created(before `init()` been called). 
Above is just guess，I will debug for it after I get a proper environment.